thorlcr/thorutil/thmem.cpp

```diff
@@ -543,39 +520,60 @@ inline const void *CThorExpandingRowArray::_allocateRowTable(rowidx_t num, unsig
     }
 }
 
-inline bool CThorExpandingRowArray::_ensure(rowidx_t requiredRows, unsigned maxSpillCost)
+inline bool CThorExpandingRowArray::doResize(void **&_rows, rowidx_t requiredRows, bool copy, unsigned maxSpillCost, memsize_t &capacity, const char *errMsg)
```

Comment:
minor: Might be worth coming up with clearer function name.  resize, _resize and doResize aren't obvious what they do.